src/SupabaseGrid.tsx

```diff
@@ -170,6 +170,7 @@ const SupabaseGridLayout = React.forwardRef<SupabaseGridRef, SupabaseGridProps>(
         <Header
           onAddRow={editable ? props.onAddRow : undefined}
           onAddColumn={editable ? props.onAddColumn : undefined}
+          editable={editable}
```

Comment:
you can get the grid editable param [from store](https://github.com/supabase/grid/blob/2c63a6fb398603b1dffa53fe2356b42810e10987/src/store/reducers/base.ts#L17). No need to pass it around.

Reply:
Oh yeah, oops, will update it.